src/Text/Pandoc/Writers/OpenDocument.hs

```diff
@@ -398,7 +414,7 @@ blockToOpenDocument o bs
                            inTags True "text:list" [ ("text:style-name", "L" <> tshow ln)]
                                       <$> orderedListToOpenDocument o pn b
       table :: PandocMonad m => Ann.Table -> OD m (Doc Text)
-      table (Ann.Table _ (Caption _ c) colspecs thead tbodies _) = do
+      table (Ann.Table (ref, _, _) (Caption _ c) colspecs thead tbodies _) = do
```

Comment:
`ref` is usually called `ident` in the code base.

Reply:
Fixed everywhere - I hope.